src/ProjectTemplates/BlazorTemplates.Tests/Helpers/Project.cs

```diff
@@ -0,0 +1,409 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
```

Comment:
Could we source include this file from the regular project template tests? This is a fair bit of code to duplicate.

Reply:
Same for the other types that look like they could be shared rather than duplicated.

Reply:
Yeah i mentioned that earlier as well, it should be as easy as moving these files to a shared source directory and including them here and in project templates

Reply:
This is in the Shared repository now.